Rubberduck.Inspections/InspectionResult.cs

```diff
@@ -65,17 +70,17 @@ protected InspectionResultBase(IInspection inspection, QualifiedModuleName quali
         /// <summary>
         /// Gets the information needed to select the target instruction in the VBE.
         /// </summary>
-        public virtual QualifiedSelection QualifiedSelection
+        public QualifiedSelection QualifiedSelection
         {
             get
             {
-                return Context == null
+                return NavigateToTarget
                     ? Target.QualifiedSelection
                     : new QualifiedSelection(QualifiedName, Context.GetSelection());
```

Comment:
Seems `QualifiedSelection` could be turned into an auto-property, with this ternary moved to the constructor.